### DISCUSSION TITLE
[Question] Is Middle East included in the scenery packages ?

### DISCUSSION THREAD

**Discussion by limkaflyer** [kubilus1/autoortho#269] [Q&A]
### question

I have the Asia package installed but when I load into Dubai, I dont see any orthos. Which exact scenery package includes the Middle East orthos ?

Thanks

**Comment by dpdurst** [comment]
I don't think Dubai is anywhere near Asia so I would expect it not to show anything. EUR or AFR would be closer but not sure it will show up in there either. Try it and let us know

**Reply by limkaflyer** [reply]
UAE is in Asia.

**Comment by Simssmith** [comment]
Dubai is part of Asia.

**Comment by dpdurst** [comment]
Yes, Dubai is in Asia, but it’s also a part of the Middle East which could also be considered part of Africa. Dubai is not a country, it is a city and emirate in a country called the United Arab Emirates, this country is in the Middle East and this is a transcontinental region, ie. it lies in both Asia and Africa and why I said it could be in the Africa scenery

**Comment by Simssmith** [comment]
Whoever is distributing world packages please use normal convention.  Entire middle east has always been considered Asia and not Europe or Africa. 

![image](https://github.com/kubilus1/autoortho/assets/26805400/ac50a480-d484-46ed-a21c-44d06362e288)
